packages/lib/services/plugins/PluginService.ts

```diff
@@ -100,6 +100,12 @@ export default class PluginService extends BaseService {
 		return this.plugins_;
 	}
 
+	public get pluginsEnabled(): Plugins {
+		const pluginSettings = this.unserializePluginSettings(Setting.value('plugins.states'));
```

Comment:
We avoid accessing settings directly from services now, so the "settings" array should be passed to this function, like it is for `pluginEnabled` for example.

Sorry could you name the function `enabledPlugins` please? The `pluginEnabled` one really should be named `pluginIsEnabled` but it's a different issue

Reply:
Actually I'm thinking of moving this piece of code into `versionInfo()`. I think it's the responsibility of `versionInfo()` to retrieve the right list of plugins, since that's the behavior we always want. Otherwise it means the caller has to decide what plugins (enabled or not) they want to list.

What do you think?

Reply:
Thanks, the tests need to be updated too

Reply:
I updated the tests.